src/components/PageLevelBanner/PageLeveBanner.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import type { StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { PageLeveBanner, Variant } from './PageLeveBanner';
+import Button from '../Button';
+
+export default {
+  title: 'Molecules/Messaging/PageLeveBanner',
+  component: PageLeveBanner,
+  args: {
+    title:
+      'New curriculum updates are available for one or more of your courses.',
+  },
+};
+
+type Args = React.ComponentProps<typeof PageLeveBanner>;
+
+const getDescription = (status?: Variant) => (
+  <>
+    Summit Learning has a full-time team dedicated to constantly improving our
+    curriculum. To see the updates,{' '}
+    <Button
+      onClick={(event: any) => event.preventDefault()}
+      status={status}
+      variant="link"
+    >
+      click into the course
+    </Button>
+    .
+  </>
+);
+
+const dismissMethod = () => {
+  console.log('dismissing~');
+};
+
+export const Brand: StoryObj<Args> = {
+  render: ({ variant, ...other }) => {
+    return (
+      <PageLeveBanner
+        description={getDescription(variant)}
+        variant={variant}
+        {...other}
+      />
+    );
+  },
```

Comment:
I think since `...Brand` is happening for every story, we can move this block into the `default` object and remove `...Brand` for the rest of the stories.

e.g.: 

```tsx
export default {
  title: 'Molecules/Messaging/PageLeveBanner',
  component: PageLeveBanner,
  args: {
    title:
      'New curriculum updates are available for one or more of your courses.',
  },
  render: ({ variant, ...other }) => {
    return (
      <PageLeveBanner
        description={getDescription(variant)}
        variant={variant}
        {...other}
      />
    );
  },
} as Meta<Args>;

...

export const Brand: StoryObj<Args> = {};

export const Success: StoryObj<Args> = {
  args: {
    variant: 'success',
  },
};

...
```

Reply:
Oh awesome, thanks!